pom.xml

```diff
@@ -111,6 +136,21 @@
       <version>26.0-jre</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.locationtech.jts</groupId>
+      <artifactId>jts-core</artifactId>
+      <version>1.15.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.locationtech.jts.io</groupId>
+      <artifactId>jts-io-common</artifactId>
+      <version>1.15.0</version>
+    </dependency>
+    <dependency>
+      <groupId>com.betadriven</groupId>
+      <artifactId>jackson-datatype-jts</artifactId>
+      <version>2.6-SNAPSHOT</version>
```

Comment:
This is still a SNAPSHOT depdendency, meaning that we would have to have this project built locally for each workstation? Where can this version be found?

Reply:
<https://github.com/ernstdevreede/jackson-datatype-jts> has to be built locally. I think this is tolerable as long as we're working only within the [feature-sigmet-model](https://github.com/fmidev/fmi-avi-messageconverter/tree/feature-sigmet-model) branch.